code/__DEFINES/is_helpers.dm

```diff
@@ -73,7 +73,7 @@ GLOBAL_LIST_INIT(turfs_without_ground, typecacheof(list(
 #define iszombie(A) (is_species(A, /datum/species/zombie))
 #define isskeleton(A) (is_species(A, /datum/species/skeleton))
 #define ismoth(A) (is_species(A, /datum/species/moth))
-#define ishumanbasic(A) (is_species(A, /datum/species/human))
+#define ishumanbasic(A) (is_species(A, /datum/species/human) && !is_species(A, /datum/species/human/krokodil_addict))
```

Comment:
?

Reply:
We dont want krokodil people to be considered basic humans. I plan on doing the same for felinids in a future PR.

Reply:
You changed them to be human subtype

Reply:
Yes. and I plan on making ishumanbasic only return true for *true humans*. Out of scope for now because felinid code is stupid bad.

Reply:
I don't believe it is out of scope if you are introducing this behavior in this PR

Reply:
Its more of a headache than you think. Ill do it in a follow up.